packages/web/src/scss/components/Accordion/_Accordion.scss

```diff
@@ -0,0 +1,90 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../tools/typography';
+
+.Accordion__itemHeader {
+    position: relative;
+    display: flex;
+    gap: theme.$accordion-header-gap;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 100%;
+    height: auto;
+    padding: theme.$accordion-header-padding-y theme.$accordion-header-padding-x;
+    margin-bottom: 0;
+    border-radius: theme.$accordion-border-radius;
+
+    &:hover {
+        background-color: theme.$accordion-item-color-hover;
+    }
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: theme.$accordion-header-padding-y;
+        display: block;
+        width: calc(100% - (#{theme.$accordion-header-padding-y} * 2));
```

Comment:
in my opinion, if you would set `right` to `padding-y` and `width` to 100% it would be more readable. But it probably doesn't matter.